### PR TITLE
Added network visualization

### PR DESCRIPTION
**Pull request by jennifer-chao**

### CODE REVIEW

File: appserver/website-bootstrap/network.html [hunk 1]
```diff
@@ -46,6 +47,156 @@ <h3>Analyze Network</h3>
...
+								{"id": "11"},
+								{"id": "12"},
+								{"id": "13"},
+								{"id": "14"},
+								{"id": "15"},
+								{"id": "16"},
+								{"id": "17"},
+								{"id": "18"},
+								{"id": "19"},
+								{"id": "20"}
```
Comment (BareBeaverBat) [inline appserver/website-bootstrap/network.html]: Would it be much trouble to rename these ids to something like "n#" (e.g. "n1", "n2", "n3", etc.)? I'm concerned that having ids that were simply numbers would have an elevated risk of causing confusion.

File: appserver/website-bootstrap/network.html [hunk 2]
```diff
@@ -46,6 +47,156 @@ <h3>Analyze Network</h3>
...
+						var simulation = d3.forceSimulation()
+							.force("link", d3.forceLink().id(function(d) { return d.id; }))
+							.force("charge", d3.forceManyBody()
+								.strength(-300)
+								.theta(0.8)
+								.distanceMax(150)
+							)
+							.force("center", d3.forceCenter(width / 2, height / 2));
+
+						const graph = {
```
Comment (BareBeaverBat) [inline appserver/website-bootstrap/network.html]: this variable name is shadowed by the parameter name inside the 'run(graph)' function. To reduce the risk of bugs, would you mind giving this variable a more specific name to differentiate it from the function parameter?

File: appserver/website-bootstrap/network.html (hunk 1 above)
Comment (jennifer-chao) [reply appserver/website-bootstrap/network.html]: you got it

File: appserver/website-bootstrap/network.html (hunk 2 above)
Comment (jennifer-chao) [reply appserver/website-bootstrap/network.html]: ah actually the parameter name is that graph object. I did rename it to graphData to make it more specific, but they're the same

File: appserver/website-bootstrap/network.html [hunk 3]
```diff
@@ -46,6 +47,156 @@ <h3>Analyze Network</h3>
...
+								{"source": "n2", "target": "n10", "value": 7},
+								{"source": "n3", "target": "n9", "value": 7},
+								{"source": "n4", "target": "n8", "value": 20},
+								{"source": "n4", "target": "n7", "value": 15},
+								{"source": "n8", "target": "n1", "value": 1},
+								{"source": "n6", "target": "n1", "value": 1}
+							]
+						}
+  
+						function run(graphData) {
```
Comment (BareBeaverBat) [inline appserver/website-bootstrap/network.html]: I might be misunderstanding Javascript syntax, but isn't this line declaring a new function which has a parameter named 'graphData'? My concern had been that the variable which was declared before the function definition had the same name as the parameter of that function.
While in this case the existing variable was then simply passed to that function as an argument (for the parameter of the same name), my concern was that the parameter & the already-existing variable having the same name might cause confusion if this code is expanded on later.

Comment (jennifer-chao) [reply appserver/website-bootstrap/network.html]: ohh my bad I misread what you sent the first time yeah I got it

Comment (BareBeaverBat) [reply appserver/website-bootstrap/network.html]: No problem. Thanks!

**Review by BareBeaverBat — APPROVED** [review]

**Pull request merged**